docker/ubuntu-22.04-amd64-openssl-1.1.x/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# syntax=docker/dockerfile:1
+# Regression test for: https://github.com/prisma/prisma/issues/25101
+
+ARG UBUNTU_VERSION="22.04"
+
+FROM ubuntu:${UBUNTU_VERSION}
+
+ARG NODE_VERSION="18.18.2"
+ARG ARCH="x64"
+
+WORKDIR /usr/src/app
+COPY --from=app . ./
+COPY --from=utils . ./
+
+# Install system dependencies for Node.js
+RUN apt-get update -y \
+  && apt-get install -y curl xz-utils
+
+# Install Node.js
+RUN curl -fsSLO --compressed "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-${ARCH}.tar.xz" \
+  && tar -xJf "node-v${NODE_VERSION}-linux-${ARCH}.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
+  && rm "node-v${NODE_VERSION}-linux-${ARCH}.tar.xz"
+
+# Install libssl1.1
+RUN curl -fsSLO http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb \
+  && dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb \
+  && rm libssl1.1_1.1.1f-1ubuntu2_amd64.deb
```

Comment:
So it's not even in the repository, curious why so many users had it on their machines.

Also what I don't quite understand is that this package comes from Ubuntu 20.04, AFAICT, and yet there are no issues there.